web-app/src/app/stitch-canvas/stitch-canvas.html

```diff
@@ -68,41 +68,51 @@
         super.connectedCallback();
 
         let position = null;
+        let drawBackstitch = true;
 
         const getPosition = event => {
           const e = (event.touches && event.touches[0] || event);
           return { x: Math.round(e.clientX), y: Math.round(e.clientY) };
         };
 
         const onStart = event => {
-          if ((event.type === "mouseenter" && event.buttons & 1 === 1) || event.type !== "mouseenter") {
```

Comment:
This logic with `mouseenter` is now broken. The idea was to handle when user moved the pointer out of view port by accident.